src/components/ComicDetail/ComicDetail.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Text} from 'react-native';
+import {ActivityIndicator, Colors} from 'react-native-paper';
+
+import {useGetComicByIdToStoriesQuery} from '../../api/comics';
+
+export default function ComicDetail({comicId, item}) {
+  const {data, error, isFetching, isLoading} =
+    useGetComicByIdToStoriesQuery(comicId);
+  if (error) return <Text>Error</Text>;
+  if (isFetching)
+    return <ActivityIndicator animating={true} color={Colors.red800} />;
+  if (isLoading)
+    return <ActivityIndicator animating={true} color={Colors.red800} />;
+  if (!data) return null;
+  if (data) {
+    console.log(data);
+    return (

```

Comment:
Kodu formatlasaydık iyiydi, anlaşılmıyor hiç bir şey. Önceki ödevden sonra belirtmiştim bunu da.
```suggestion
export default function ComicDetail({comicId, item}) {
  const {data, error, isFetching, isLoading} =
    useGetComicByIdToStoriesQuery(comicId);
  
  if (error) {    
      return <Text>Error</Text>;
  }
  
  if (isFetching) {
    return <ActivityIndicator animating={true} color={Colors.red800} />;
  }
  
  if (isLoading){
    return <ActivityIndicator animating={true} color={Colors.red800} />;
   }
  
  if (!data) {
    return null;
  }
  
  if (data) {
    console.log(data);
    return (
```
Ek olarak belirteyim, isFetching ve isLoading aynı işi yapıyor. Son if olan `if(data)` kısmına da gerek yok. Zaten başka ihtimal kalmıyor, direkt return edilebilirdi.